### PR TITLE
releases: Update patch release schedule and add Release Manager info

### PR DESCRIPTION
**Pull request by justaugustus**
Continuation of @jimangel's wonderful work in https://github.com/kubernetes/website/pull/27929.

- OWNERS: Update SIG Release content reviewers/approvers
- scripts: Allow SIG Release to approve changes to update-release-info
- releases: Canonicalize patch release information on k/website
- releases: Canonicalize Release Managers page on k/website
- release-managers: Update personnel
- release-managers: Link cleanup
- patch-releases: Update cycle after May releases and EOL 1.18 (supersedes https://github.com/kubernetes/sig-release/pull/1570, FYI @cpanato)

Signed-off-by: Stephen Augustus <foo@auggie.dev>

/hold for:
- [x] discussion
- [x] approval of future patch schedule
- [x] accompanying tombstones in k/sig-release: https://github.com/kubernetes/sig-release/pull/1577
- [x] add `area/release-eng` label to k/website: https://github.com/kubernetes/test-infra/pull/22272

cc: @kubernetes/sig-release-leads @kubernetes/release-engineering

### CODE REVIEW

**Comment by jimangel** [comment]
/lgtm
/approve

Good call on moving the script, I look forward to the day we delete it all together! I think everything in this PR is fine as we continue iterating on what the releases page looks like.

Is the schedule still pending?

**Comment by justaugustus** [comment]
> Good call on moving the script, I look forward to the day we delete it all together!

Hehe, I was considering removing in this pass, but I want to noodle more on where the last page should go, since it's currently in k/community.

Ultimately, it may end up being a better fit in k8s.dev.

> Is the schedule still pending?

Yep. We'll let @kubernetes/release-engineering review next week before merging.

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by irvifa — COMMENTED** [review]
/lgtm

File: content/en/releases/OWNERS
Comment (sftim) [inline content/en/releases/OWNERS:17]: We might need to add this label to the repo as well(?)

File: content/en/releases/release-managers.md
Comment (sftim) [inline content/en/releases/release-managers.md:4]: Is this original documentation or duplicated from elsewhere? (It looks like it'd be a better fit for https://k8s.dev/)

File: scripts/releng/update-release-info.sh
Comment (sftim) [inline scripts/releng/update-release-info.sh:7]:
```suggestion
title: Kubernetes Release Cycle
```

File: content/en/releases/patch-releases.md
Comment (xmudrii) [inline content/en/releases/patch-releases.md:84]: Do we have some rules that we're following for choosing the release day? I'm mostly wondering about August, since Wednesday after the 11th is the 18th, which seems like a reasonable release date as well. IIRC, we did releases on and around the 18th of the month in the past.

File: scripts/releng/update-release-info.sh
Comment (xmudrii) [inline scripts/releng/update-release-info.sh:4]: What about other documents -- `patch-releases.md` and `schedule.yaml`? Are we permanently moving those two documents to this repo?

File: scripts/releng/update-release-info.sh
Comment (wilsonehusin) [inline scripts/releng/update-release-info.sh:11]:
```suggestion
<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/releng/update-release-info.sh in k/website -->
```

File: content/en/releases/patch-releases.md
Comment (saschagrunert) [inline content/en/releases/patch-releases.md:12]: I assume we deprecate https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md and link it into this location, right?

Comment (sftim) [reply content/en/releases/patch-releases.md:12]: If we edit https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md to mention deprecation or a link to the canonical page, we'll need to automate _removing_ that deprecation text / link from the copy on https://kubernetes.io/

Comment (sftim) [reply content/en/releases/patch-releases.md:12]: (I'm fine with doing that, but it'd need to happen)

File: content/en/releases/patch-releases.md
Comment (sftim) [reply content/en/releases/patch-releases.md:84]: I don't think those rules would be part of this PR.

File: content/en/releases/patch-releases.md
Comment (justaugustus) [reply content/en/releases/patch-releases.md:12]: @sftim @saschagrunert -- That's the plan! (mentioned in the PR description)

>     * releases: Canonicalize patch release information on k/website
>     * releases: Canonicalize Release Managers page on k/website

> /hold for:
> 
>     * [ ]  discussion
>     * [ ]  approval of future patch schedule
>     * [ ]  accompanying tombstones in k/sig-release

The script has already been updated in support of that (ref: https://github.com/kubernetes/website/pull/27997/commits/39796d148141728307f3bcdd5f7e149a596b80a3).
Tombstones to follow.

**Comment by justaugustus** [comment]
@kubernetes/sig-release-leads @sftim --

I've added tombstones to de-dupe the page content here: https://github.com/kubernetes/sig-release/pull/1577
Will follow-up with the `area/release-eng` later today.

@wilsonehusin -- Nice catch on script location!

@kubernetes/release-engineering -- How are we feeling about the proposed dates?

All -- anything else that needs review here?

File: scripts/releng/update-release-info.sh
Comment (justaugustus) [reply scripts/releng/update-release-info.sh]: Fixed!

File: scripts/releng/update-release-info.sh
Comment (justaugustus) [reply scripts/releng/update-release-info.sh]: Fixed.

File: content/en/releases/OWNERS
Comment (justaugustus) [reply content/en/releases/OWNERS:17]: PR opened for the `area/release-eng` label here: https://github.com/kubernetes/test-infra/pull/22272

File: content/en/releases/release-managers.md
Comment (justaugustus) [reply content/en/releases/release-managers.md:4]: @sftim -- some of the other pages link out to the Release Managers contact info, so it feels right to keep them close.

The contact info also is needed by both contributors and external consumers, which makes it a better fit for k/website.

File: content/en/releases/patch-releases.md
Comment (justaugustus) [reply content/en/releases/patch-releases.md:84]: > I don't think those rules would be part of this PR.

Agreed.
@xmudrii -- would you like to own discussing a policy as a follow-up action?

File: scripts/releng/update-release-info.sh
Comment (justaugustus) [reply scripts/releng/update-release-info.sh:4]: Yep, there's an accompanying PR in k/sig-release here: https://github.com/kubernetes/sig-release/pull/1577

**Review by cpanato — APPROVED** [review]
dates lgtm
and thanks for all this!
/lgtm

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by justaugustus** [comment]
Thanks everyone!
/hold cancel

File: content/en/releases/patch-releases.md
Comment (xmudrii) [reply content/en/releases/patch-releases.md:84]: @justaugustus -- Sounds good. I'll take care of that.

**Review by xmudrii — APPROVED** [review]
/lgtm

**Pull request merged**
